doc/changelog.rst

```diff
@@ -89,6 +89,14 @@ Changes in Version 4.0
 
 .. warning:: PyMongo 4.0 drops support for MongoDB 2.6, 3.0, 3.2, and 3.4.
 
+.. warning:: PyMongo 4.0 changes the default value of the ``directConnection`` URI option and
+  keyword argument to :class:`~pymongo.mongo_client.MongoClient`
+  to ``False`` instead of ``None``, allowing for the automatic
+  discovery of replica sets. This means that if you
+  want a direct connection to a single server you must pass
+  ``directConnection=True`` as a URI option or keyword argument.
+  See the :doc:`migrate-to-pymongo4` for more details.
```

Comment:
Can we link to the directConnection section in the migration guide?

Reply:
Done.